src/detect-engine.c

```diff
@@ -117,12 +117,10 @@ SCEnumCharMap det_ctx_event_table[] = {
     { "Z_STREAM_ERROR", FILE_DECODER_EVENT_Z_STREAM_ERROR },
     { "Z_BUF_ERROR", FILE_DECODER_EVENT_Z_BUF_ERROR },
     { "Z_UNKNOWN_ERROR", FILE_DECODER_EVENT_Z_UNKNOWN_ERROR },
+    { "LZMA_IO_ERROR", FILE_DECODER_EVENT_LZMA_IO_ERROR },
```

Comment:
These needs new events in rules/http-events.rules

Reply:
I see that all of the rules in `rules/http-events.rules` are triggering off of the events defined in `app-layer-htp.c::http_decoder_event_table`. These events here for lzma decompression are defined in `detect-engine.c` and none of them have corresponding signatures in `http-events.rules`, and I don't see a corresponding `.rules` file that is matching on these file events.

Do you want me to define a new `HTTP_DECODER_EVENT`, and then set it in `util-file-swf-decompression.c` whenever we have `FILE_DECODER` events (and do we want to match all of them, or just some of them?), or do you want me to create new rules in `http-events.rules` that trigger on `FILE_DECODER` events? Or something else? I am just not clear what the preferred way to go about this is.

Reply:
FYI, there is a known issue with these events, see https://redmine.openinfosecfoundation.org/issues/4482 (@jlucovsky seems I lost track of these)

Reply:
Ack. So then fixing the file decoder events logging, and making rules to trigger on them, is a separate ticket / PR?

Reply:
Yes